src/mpv_calendar/server.py

```diff
@@ -300,6 +316,7 @@ async def add_playlist_item(request: Request) -> dict[str, str]:
     "index",
     "next_playlist_item",
     "previous_playlist_item",
+    "queue_playlist_item",
```

Comment:
Implement the concept of queuing vs. adding. Adding is a misnomer here, it just plays it instantly. Queuing plays it when the current video finishes or when the "Next" button is pressed.